components/monitoring/prometheus/stone-stg-m01/monitoring-prometheus-oauth-config-path.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- op: add
```

Comment:
folder for specific cluster should go under a folder for an environment, for example
`components/gitops/staging/stone-stg-m01/`